handbook/strategy/recurring-processes.md

```diff
@@ -18,6 +18,42 @@ At the end of a school year (towards the end of the winter semester), the Strate
 
 The [club objectives page](/handbook/strategy/objectives.md) should be updated with our progress, and improvements we could make to our processes should be updated throughout the handbook. An announcement should be made in Slack about our goals, any outcomes from the retrospective, and whether the club has any feedback for the leadership team.
 
+## Kickoff Event
+
+Each school year starts with a kickoff event where new members are introduced to Launch Pad and project ideation happens. Smaller-scale versions of this event might happen at the start of the second semester, depending on how many new projects are started.
+
+### Schedule
+
+This schedule might vary year to year, but kickoff events will generally follow this format.
+
+| Time | Activity
+| ---- | --------
+| 30m  | Presentation: "Welcome to Launch Pad"
+| 15m  | Icebreaker: Round 1
+| 15m  | Icebreaker: Round 2
+| 15m  | Break
+| 10m  | Presentation: "Project Ideation" - see [Ideation](#ideation)
+| 15m  | Ideation: Round 1
+| 5m   | Break and ideas updates
+| 15m  | Ideation: Round 2
+| 5m   | Break and ideas updates
+| 15m  | Ideation: Round 3
+| TODO | TODO
+
+### Ideation
+
+Project ideation revolves around the guidelines provided in our [Project Scope guide](/handbook/project-management/scope.md), especially our guidelines about using a "core feature" to help narrrow MVP scope.
+
+In general, the process is as follows:
+
+1. Prior to kickoff, we will ask for some "core feature" submissions via Google Forms from the entire club
+2. Leads will review the submissions and consolidate them into "ideas", with each lead adopting one idea. These will ask as discussion points.
+3. During kickoff, the "ideas" will be presented to the club, and we will form discussion groups based on interest to refine each "idea" back down into a "core feature". Discussions will take place over several iterations.
+   * At each iteration, leads will provide updates on where each "idea" is going. In between iterations, members will be able to move between groups based on what they might be interested in.
+4. After the last round of discussions, the groups will become Launch Pad project teams!
```

Comment:
What happens if there are too many people in one group?

Reply:
In my mind we'll just have to figure out a way to rebalance groups manually and ask folks nicely if they would be okay with switching :( I'll add a point here, but do you have another idea?

Reply:
Hmm. Last year we had a google form sent out, ranking our top 3 teams. I think everyone got their first choice anyways so it didn't matter.

Reply:
I couldn't think of a better way to go about this, so let's just do that again 95f1dd0f1a1e72cce6a733180e5f9806926e8f2b

Reply:
What if we ask people to form equal groups first then have them ideate as a group?

Reply:
It makes it very difficult to pull off any ideation at all (we effectively used to do this with platform teams), since the team starts either at 0 or at a point where a few members might be meh about the idea, and half of those will be too pressured by the initial idea to change it drastically

Reply:
ohh I see, alright that makes sense